Geometry/ForwardGeometry/src/ZdcHardcodeGeometryLoader.cc

```diff
@@ -34,6 +34,7 @@ ZdcHardcodeGeometryLoader::load(DetId::Detector /*det*/, int subdet)
       fill(HcalZDCDetId::EM  ,hg );
       fill(HcalZDCDetId::LUM ,hg );
       fill(HcalZDCDetId::HAD ,hg );
+//      fill(HcalZDCDetId::RPD ,hg );
```

Comment:
@kpedro88 - it is advisable to check a geometry handle validity before accessing it (unrelated to where it comes from)